### PR TITLE
Fixed mistakes pointed out in #220.

### PR DESCRIPTION
**Pull request by m1cm1c**

### CODE REVIEW

**Comment by jGleitz** [comment]
This is ***not*** the correct fix. Please only change `\glslabel` -> `\glslink` and everything will work as intended.

**Comment by jGleitz** [comment]
No! You should change `\glslabel` -> `\glslink` from the **initial** state:

```LaTeX
\glslink{software architecture}{architecture}
```

```LaTeX
\glslabel{software architecture}{architectural}
```

(sorry for the confusion, my comment was not precise)

**Comment by m1cm1c** [comment]
So there now should be `\glslink{software architecture}{architectural}` in the file?

**Comment by jGleitz** [comment]
> So there now should be \glslink{software architecture}{architectural} in the file?

Yep that means “print ‘architectural’ but link it to ‘software architecture’”

**Comment by m1cm1c** [comment]
Okay, good to know. I assumed this was an ugly artefact resulting from automated merging.

File: doc/Requirements Specification/Parts/Models.lyx [hunk 1]
```diff
@@ -674,8 +674,8 @@ glspl{component}
  They soon realise that there are multiple ways to implement the conversion
  engine.
  Especially, there are different ways to parallelise the task.
- Different developers come up with different approaches, all having there
- pro and contra arguments.
+ Different developers come up with different approaches, all having their
+ advatages and disadvantiges.
```
Comment (jGleitz) [inline doc/Requirements Specification/Parts/Models.lyx]: “disadvantages”

File: doc/Requirements Specification/Parts/Models.lyx [hunk 2]
```diff
@@ -179,7 +179,7 @@ status open
 
 
 \backslash
-glslabel{software architecture}{architecture}
+gls{software architecture}
```
Comment (jGleitz) [inline doc/Requirements Specification/Parts/Models.lyx]: I’d prefer `\glslink{software architecture}{architectural}` here to not say “software” two times.

File: doc/Requirements Specification/Parts/Models.lyx (hunk 1 above)
Comment (m1cm1c) [inline doc/Requirements Specification/Parts/Models.lyx]: Thank you.

File: doc/Requirements Specification/Parts/Models.lyx (hunk 2 above)
Comment (m1cm1c) [inline doc/Requirements Specification/Parts/Models.lyx]: You mean `\glslink{software architecture}{architecture}`, don't you? I changed it to this.

**Comment by RomanLangrehr** [comment]
Lgtm

**Pull request merged**